code/game/objects/random/mechs.dm

```diff
@@ -0,0 +1,40 @@
+/obj/random/mech
+	name = "random mech"
+	desc = "This is a random single mech."
+	icon = 'icons/mecha/mecha.dmi'
+	icon_state = "old_durand"
+	drop_get_turf = FALSE
+
+//This list includes the phazon, gorilla and mauler. You might want to use something else if balance is a concern.
+/obj/random/mech/item_to_spawn()
+	return pick(/obj/mecha/combat/gygax,
+				/obj/mecha/combat/gygax/serenity,
+				/obj/mecha/combat/gygax/dark,
+				/obj/mecha/combat/marauder,
+				/obj/mecha/combat/marauder/seraph,
+				/obj/mecha/combat/marauder/mauler,
+				/obj/mecha/medical/odysseus
+				/obj/mecha/combat/phazon,
+				/obj/mecha/combat/phazon/janus,
+				/obj/mecha/combat/durand,
+				/obj/mecha/working/ripley,
+				/obj/mecha/working/ripley/firefighter,
+				/obj/mecha/working/ripley/deathripley,
+				/obj/mecha/working/ripley/mining)
+
+
+/obj/random/mech/old
+	name = "random mech"
+	desc = "This is a random single old mech."
+	icon = 'icons/mecha/mecha.dmi'
+	icon_state = "old_durand"
+	drop_get_turf = FALSE
+
+//Note that all of those are worn out and have slightly less maximal health than the standard.
+/obj/random/mech/old/item_to_spawn()
```

Comment:
Marauder / phazon in this list is a bit concerning to me, since they will still spawn with equipment.

Reply:
That's the base list, here for the sake of having one already made. 
Easy enough to make another for future, specific applications.

Reply:
Fixed in commit.